common/djangoapps/custom_reports/views.py

```diff
@@ -0,0 +1,29 @@
+"""
+Reports view functions
+"""
+from django.http import HttpResponse
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+
+
+# sample callback
+@csrf_exempt
+@require_POST
+def my_callback(request):

```

Comment:
I think this method has to be redefined, what is the purpose of this ?

Reply:
Ah, it's a sample callback to show how the web hook would work. I think we can safely ignore it?